contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -4229,3 +4229,37 @@ BEGIN
             RETURN NULL;
 END; $BODY$
 LANGUAGE plpgsql;
+
```

Comment:
if this is for ocelot, file should change to 2.3 to 2.4

Reply:
done